packages/react-reconciler/src/ReactFiberLane.new.js

```diff
@@ -458,8 +458,15 @@ export function includesNonIdleWork(lanes: Lanes) {
 export function includesOnlyRetries(lanes: Lanes) {
   return (lanes & RetryLanes) === lanes;
 }
-export function includesOnlyTransitions(lanes: Lanes) {
-  return (lanes & TransitionLanes) === lanes;
+export function includesOnlyTransitionsOrHydration(lanes: Lanes) {
+  const TransitionOrHydrationLanes =
+    TransitionLanes |
+    InputContinuousHydrationLane |
+    DefaultHydrationLane |
+    TransitionHydrationLane |
+    SelectiveHydrationLane |
+    IdleHydrationLane;
+  return (lanes & TransitionOrHydrationLanes) !== NoLanes;
```

Comment:
This needs to be `=== lanes` like before in case the transition got entangled with a non-urgent update for some reason